### PR TITLE
Emergency energy shield balloon alert

### PR DESCRIPTION
**Pull request by RaveRadbury**
## About The Pull Request

Replaces the to_chat securing messages for the Emergency Energy Shield with balloon_alert

## Why It's Good For The Game

It's a good candidate for balloon alerts, less chat spam.

## Changelog

:cl:
qol: Added balloon alerts for securing emergency energy shields
/:cl:

### CODE REVIEW

File: code/game/machinery/shieldgen.dm [hunk 1]
```diff
@@ -201,11 +201,11 @@
 		return
 	if(!anchored && !isinspace())
 		tool.play_tool_sound(src, 100)
-		to_chat(user, span_notice("You secure \the [src] to the floor!"))
+		balloon_alert(user, span_notice("secured."))
```
Comment (tralezab) [inline code/game/machinery/shieldgen.dm:204]:
```suggestion
		balloon_alert(user, span_notice("secured"))
```

File: code/game/machinery/shieldgen.dm [hunk 2]
```diff
@@ -201,11 +201,11 @@
...
 		set_anchored(TRUE)
 	else if(anchored)
 		tool.play_tool_sound(src, 100)
-		to_chat(user, span_notice("You unsecure \the [src] from the floor!"))
+		balloon_alert(user, span_notice("unsecured."))
```
Comment (tralezab) [inline code/game/machinery/shieldgen.dm:208]:
```suggestion
		balloon_alert(user, span_notice("unsecured"))
```

**Review by tralezab — CHANGES_REQUESTED** [review]
surprisingly, the balloon alert standard has no punctuation outside of "..." for waiting like do afters and "!" for failures

**Comment by tralezab** [comment]
thank you a ton though

File: code/game/machinery/shieldgen.dm [hunk 3]
```diff
@@ -201,11 +201,11 @@
 		return
 	if(!anchored && !isinspace())
 		tool.play_tool_sound(src, 100)
-		to_chat(user, span_notice("You secure \the [src] to the floor!"))
+		balloon_alert(user, span_notice("secured"))
```
Comment (Mothblocks) [inline code/game/machinery/shieldgen.dm:204]:
```suggestion
		balloon_alert(user, "secured")
```

File: code/game/machinery/shieldgen.dm [hunk 4]
```diff
@@ -201,11 +201,11 @@
...
 		set_anchored(TRUE)
 	else if(anchored)
 		tool.play_tool_sound(src, 100)
-		to_chat(user, span_notice("You unsecure \the [src] from the floor!"))
+		balloon_alert(user, span_notice("unsecured"))
```
Comment (Mothblocks) [inline code/game/machinery/shieldgen.dm:208]:
```suggestion
		balloon_alert(user, "unsecured")
```

**Review by Mothblocks — CHANGES_REQUESTED** [review]

**Pull request merged**